hs_core/hydroshare/utils.py

```diff
@@ -1104,3 +1104,39 @@ def resolve_request(request):
         return request.data
 
     return {}
+
+
+def format_datetime(dt, template='{0.year:4d}-{0.month:02d}-{0.day:02d}'):
```

Comment:
Since moving to named templates, I think the default should follow the same pattern by using a template label rather than the value.